src/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+include(GNUInstallDirs)
+
+add_subdirectory(fields)
+
+add_compile_definitions($<$<BOOL:${HAVE_CLZ}>:HAVE_CLZ>)
```

Comment:
Instead of attaching CLMUL to the fields interface, how about handling it here?
```CMake
set_source_files_properties(minisketch.cpp PROPERTIES COMPILE_DEFINITIONS $<$<BOOL:${HAVE_CLMUL}>:HAVE_CLMUL>)
```

Reply:
The idea was to keep all `HAVE_CLMUL`-related code local.

Reply:
Thanks! Your suggestion has been taken with reordering to to keep code specific to each target as local as possible.